tests/jobs/test_api.py

```diff
@@ -293,17 +294,14 @@ async def test_ok(self, fake2: DataFaker, spawn_job_client):
             seconds=1
         )
 
-    async def test_not_found(self, spawn_client: ClientSpawner):
+    async def test_not_found(self, snapshot, spawn_client: ClientSpawner):
```

Comment:
This is muddling things a bit.

This test should use `spawn_job_client`. This is a mistake in how the test was written and is the reason `404: Not Found` is being returned in `message`. The path is not actually configured in the router because `/jobs/:id/ping` is only available on the jobs API. This `404` is coming from the router instead of the request handler.

Please revert changes to this test and change `spawn_client` to `spawn_job_client`.